### PR TITLE
Change AF to allow non-member submissions that go into a review queue

### PR DESCRIPTION
**Pull request by darkruby501**
This PR make it so that:
* Non AF members can submit posts and comment on the AF forum
* When they click on comment boxes and new post buttons, there is a popup explaining non-member posting. This popup will display each time until they click "I understand (Don't show again').
* Upon submission of a post or comment, they receive another popup explaining that it has been added to a review queue but will appear immediately on LessWrong. These popups always display on submission.
* A non-member submission adds posts and comments to "Suggested Comments/Posts" in the Sunshine SIdebar.
* The suggested items in the Sunshine Sidebar now have a button for conveniently messaging users (so that they can be given feedback upon rejection).

-------
Initial popups will be triggered by clicking in any of the following places:
- The New Post and New Question buttons in the user menu.
- The comment box on post pages.
- Comment boxes in Recent Discussion
- Comment and Answer boxes on Question pages.
- The text field on new post pages.
Again, they won't appear after a user dismisses them, but doing this ensures they see what the process is before they write and submit content.

*Note that I have not enabled shortform UI for non-members. 
* Also, logged out members who would get login/signup popups still get those, not the new non-AF member popups.
-------
## UI shots

User Menu for non-AF Members
 
image

AI Alignment Forum 2021-07-30 13-57-45

AI Alignment Forum 2021-07-30 14-01-17

AI Alignment Forum 2021-07-30 14-05-28

image

### CODE REVIEW

**Comment by darkruby501** [comment]
Also there's this: 
Values Form a Shifting Landscape (and why you might care) - AI Alignment Forum 2021-08-02 19-07-55

**Comment by darkruby501** [comment]
Revised styling:

image

File: packages/lesswrong/lib/alignment-forum/displayAFNonMemberPopups.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import {userNeedsAFNonMemberWarning} from "./users/helpers";
+import {commentSuggestForAlignment} from "./comments/helpers";
+import {postSuggestForAlignment} from "./posts/helpers";
+import {OpenDialogContextType} from "../../components/common/withDialog";
+
+
+const isComment = (document: PostsBase | CommentsList) : document is CommentsList => {
+  if (document.hasOwnProperty("postId")) return true
```
Comment (Discordius) [inline packages/lesswrong/lib/alignment-forum/displayAFNonMemberPopups.ts:8]: Some comments are on tags. So they also won't have this.

**Comment by darkruby501** [comment]
Test of Approval Message - AI Alignment Forum 2021-08-10 10-03-54

Adding in this message now that we'll let people view their posts on AF before approval.

**Comment by darkruby501** [comment]
AI Alignment Forum 2021-08-11 16-40-56
Fixed so that notification count is in white and is visible on AF.

**Pull request merged**